README.md

```diff
@@ -2,6 +2,16 @@
 This is HACBS-test team repository.
 Purpose of this repo is to store resources, currently for tekton tasks and pipelines.
 
+## Prerequisite
```

Comment:
Should this list go under the `Policy Integration Testing` section? Only `conftest` and (arguably) `jq` are really required to run the policies themselves locally.

Reply:
@srivickynesh thanks for the change, your approach makes even more sense